doc.css

```diff
@@ -0,0 +1,359 @@
+}
+.sidebar {
+  position: sticky;
+  top: 7vh;
+  z-index: 1;
+  height: 93vh;
+  overflow-y: scroll;
+  grid-area: sidebar;
+}
+.sidebar::-webkit-scrollbar {
+  width: 0.75rem;
+}
+
+/* Track */
+.sidebar::-webkit-scrollbar-track {
+  background: var(--primaryBgDark);
+}
+
+/* Handle */
+.sidebar::-webkit-scrollbar-thumb {
+  background: var(--primaryScroll);
+  /* border-radius: 1rem; */
+}
+
+/* Handle on hover */
+.sidebar::-webkit-scrollbar-thumb:hover {
+  background: var(--secondary);
+}
+.sidebar::-webkit-scrollbar-track:hover {
+  background-color: var(--secondaryLight);
+}
+.scrollContainer {
+  padding: 2rem;
+}
+.scrollContainer > h2 {
+  font-size: 1.65rem;
+  font-weight: 700;
+  color: var(--primaryLight);
+  padding: 2rem 0rem;
+  margin: 0;
+}
+
+.scrollContainer > ul > li {
+  text-decoration: none;
+  list-style: none;
+  font-size: 1.4rem;
+  font-weight: 500;
+  color: var(--primaryLight);
+  padding: 0rem;
+  border-radius: 0.5rem;
+  transition: color 0.25s ease-in-out;
+  transition: background-color 0.25s ease-in-out;
+}
+.scrollContainer > ul > li > a {
+  display: block;
+  text-decoration: none;
+  color: var(--primaryLight);
+  padding: 1rem;
+}
+
+.scrollContainer > ul > li > a:hover {
+  color: var(--secondary);
+  cursor: pointer;
+}
+
+.scrollContainer > ul > li:hover {
+  background-color: var(--secondaryLight);
+  color: var(--secondary);
+  border-radius: 0.5rem;
+  cursor: pointer;
+}
+
+.mainContentContainer {
+  scroll-behavior: smooth;
+  position: sticky;
+  top: 7vh;
+  z-index: 1;
+  height: 93vh;
+  overflow-y: scroll;
+  padding: 2rem 4rem;
+  grid-area: mainContentContainer;
+}
+
+.mainContentContainer::-webkit-scrollbar {
+  width: 0.75rem;
+}
+
+/* Track */
+.mainContentContainer::-webkit-scrollbar-track {
+  background: var(--primaryBgDark);
+}
+
+/* Handle */
+.mainContentContainer::-webkit-scrollbar-thumb {
+  background: var(--primaryScroll);
+
+  /* border-radius: 1rem; */
+}
+
+/* Handle on hover */
+.mainContentContainer::-webkit-scrollbar-thumb:hover {
+  background: var(--secondary);
+}
+.mainContentContainer::-webkit-scrollbar-track:hover {
+  background-color: var(--secondaryLight);
+}
+
+.container > h1 {
+  color: var(--primary);
+  font-size: 3rem;
+  font-weight: 800;
+  padding: 0.5rem 0rem;
+  margin: 0;
+}
+.container > h2 {
+  color: var(--primary);
+  font-size: 2rem;
+  font-weight: 700;
+  padding: 2rem 0rem 0.5rem 0rem;
+  margin: 0;
+}
+.container > h3 {
+  font-size: 2rem;
+  font-weight: 500;
+  color: var(--primaryLight);
+  padding: 0.5rem 0rem 0.5rem 0rem;
+  margin: 0;
+}
+.container > p {
+  font-size: 1.75rem;
+  font-weight: 500;
+  color: var(--primaryLight);
+  padding: 0rem 0rem 0.5rem 0rem;
+  margin: 0;
+}
+.content {
+  border: 0.2rem solid var(--divider);
+  border-radius: 0.5rem;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+.alignCenter {
+  align-items: center;
+}
+#vh50 {
```

Comment:
Not a good practice to use the IDs as CSS selectors to style
Reason IDs should be unique on a page, override class-based styles, specificity issues, no possibility of reuse